project-p1-first-webpage-master/code/style.css

```diff
@@ -0,0 +1,223 @@
+/* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
+@import url('https://fonts.googleapis.com/css2?family=Epilogue:wght@200;500;600;800&display=swap');
+
+body {
+  background-color:#fef5e4;
+  font-family:  'Epilogue', sans-serif;
+  font-size: 22px;
+  text-align: center;
+  line-height: 200%;
+  position: absolute;
+  margin-top: 0%;
+}
+* {
+  box-sizing: border-box;
+}
+
+
+/* Formatting Header
+*/
+.Top-Box {
+  background-image: url(cover.png);
+  background-size: cover;
+  margin-top: 0%;
+  padding-bottom: 50%;
+ }
+ 
+ h1 {
+ font-family: 'Epilogue', sans-serif;
+ font-size: 100px;
+ color: #faf8f8;
+ font-style: normal ;
+ text-align: center;
+ }
+
+ /*Formatting h2*/
+ h2 {
+   text-align: center;
+   font-size: 35px;
+   margin-top: 20px;
+   padding-top: 20px;
+   font-weight: 400;
+   line-height: 100%;
+ }
+
+ h3 {
+  text-align: center;
+  font-size: 25px;
+  font-weight: 300;
+  margin-top: 20px;
+}
+
+p {
+  font-style: normal;
+  font-weight: 200;
+}
+/*Evtl wegnehmen, das sollte nicht so sein
+width:auto;
+padding: 20px;
+margin: 20px;
+border-radius: 10px;
+
+/* Formatting A1*/
+.A1 {
+    background-color: whitesmoke;
+    width:auto;
+    height: 360px;
+    padding: 0px;
+    border-radius: 10px;
+}
+
+.T1 {
+  text-align: center;
+  
+}
+
+/* Formatting A2*/
+.A2 {
+  background-image: url(lachen.jpg);
+  background-size: cover;
+  padding: 50px;
+  height: 360px;
+  opacity: 80%;
+}
+
+.T2 {
+  font-size: 90px;
+  font-weight:bolder;
+  color: #FFA476;
+  text-align: center; 
+  vertical-align: 50%;
+}
+
+
+/* Formatting A3*/
+.A3 {
+  background-color: #FFA476 50;
+  width:auto;
+  padding: 50px;
+  height: 360px
+  border-radius: 10px;
```

Comment:
Looks like you forgot the ; after `height: 360px` so the next line with `border-radius` doesn't work.